src/instructlab/sdg/default_flows.py

```diff
@@ -225,8 +226,9 @@ def get_flow(self) -> list:
                 "block_config": {
                     "block_name": "filter_relevancy",
                     "filter_column": "score",
-                    "filter_value": "2",
+                    "filter_value": "2.0",
                     "operation": operator.eq,
+                    "convert_dtype": float,
```

Comment:
This change is somehow breaking the knowledge generation. The `filter_relevancy` block is returning a dataset with 0 rows (probably because it filters out all the responses generated by the previous block). Is this expected? Same goes for `filter_verify_question` block below.

Reply:
Ah this could be because the file `src/instructlab/sdg/configs/knowledge/simple_generate_qa.yaml` still contains 

```
  {question_1}
  {response_1}

  {question_2}
  {response_2}

  {question_3}
  {response_3}
  
```
instead of 

```
  {icl_query_1}
  {icl_response_1}
 
  {icl_query_2}
  {icl_response_2}

  {icl_query_3}
  {icl_response_3}
```